src/collective/volto/formsupport/adapters/post.py

```diff
@@ -68,12 +80,24 @@ def extract_data_from_request(self):
 
         return form_data
 
-    def get_block_data(self, block_id):
+    def get_block_data(self, block_id, global_form_id):
```

Comment:
probably you don't need to pass global_form_id as parameter because is already set in __init__